### ISSUE TITLE
Custom search with GUID

### ISSUE THREAD

**Issue by hgmauri** [Alirezanet/Gridify#2]
Hi

Is there any limitation in using Gridify in Guid property in .Net Core? Because today when I make a query applying the feint informing a Guid correctly, the data is successfully returned to me, but if I inform a "broken" Guid it ignores it and returns all the records:

Example:

`http://localhost:5001/api/v1/employee/custom-search?Filter=id==DD0CB126-829F-42FB-B033-F45B2443918E`

It works correctly, but if I pass an ID with the broken guid, it is ignored and returns all records, ex:

`http://localhost:5001/api/v1/employee/custom-search?Filter=id==DD0CB126-829F-42FB-`

**Comment by Alirezanet** [comment]
Hello 
you are right. what is your expected behavior in this case ? should the library throw an exception or return null ?

**Comment by hgmauri** [comment]
I think that in this case it should always return null or empty, never return all records.

In my view, it should look like this in your code:

```
             if (queryExpression! = null)
                 query = query.Where(queryExpression);
             else
                 query = query.Take(0);
             return query;
```

**Comment by Alirezanet** [comment]
fixed in 1.1.1 version. good luck.

**Comment by Alirezanet** [comment]
please try version 1.1.2. I've added GUID handling part to manage must possible scenarios . thank you for the feedback.
